compiler/optimizations/optimizeForallUnorderedOps.cpp

```diff
@@ -242,6 +267,9 @@ void MarkOptimizableForallLastStmts::markLoopsInForall(ForallStmt* forall) {
   bool addNoTaskPrivate = forallNoTaskPrivate(forall);
   std::vector< std::vector<Expr*> > lastStatementsPerBody;
 
+  if (forall->getFunction() && forall->getFunction()->id == 1913368)
+    gdbShouldBreakHere();
```

Comment:
Is this supposed to still be here?

Reply:
@cassella - thanks for noticing, #15293 will fix it.